Guides/Combinations.md

```diff
@@ -86,7 +86,7 @@ call to `CombinationsSequence.Iterator.next()` is an O(_n_) operation.
 ### Naming
 
 The parameter label in `combination(ofCount:)` is the best match for the
-Swift API guidelines. A few other options were considered:
+[Swift's API Design Guidelines](https://www.swift.org/documentation/api-design-guidelines/). A few other options were considered:
```

Comment:
I'm fine with this change and the previous change, but they should be in a separate PR.

Reply:
Getting one PR merged has been difficult enough, I'm apprehensive to open a second just for something so tiny and low-value 😅

Reply:
I am happy to rubber-stamp that one immediately =)

Reply:
Done! https://github.com/apple/swift-algorithms/pull/210

Could you please review this PR next? :D

Reply:
Yeah, @natecook1000 and I will review this week